### PR TITLE
Theramore mechanic

### PR DESCRIPTION
**Pull request by sumankun**
## Changelog:
- Theramore mechanic with a game rule to turn it off completely

Includes:
- Two parties (xenophilic and xenophobic)
- ~25-30 different tasks. The type of the next task depends on the ratio of parties to each other
- You have rewards for completing tasks. And you can choose it. There is no penalty for not doing a task
- 4 type of policies (union, trade, culture, faith). 3 laws on each of them
- Settlers from Azeroth. Every year some random amount arrives. You also get them for completing task. Some policies also increase amount of them yearly. You can use settlers to press one of four buttons.

- New Jaina dna
- some random fixes for non-pr related stuff

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`

### CODE REVIEW

File: localization/english/wc_game_rules_l_english.yml [hunk 1]
```diff
@@ -53,4 +53,10 @@
...
  setting_wc_arthas_theramore_desc:0 "Arthas (or his alternative) will leave for Kalimdor."
 
- game_rule_category_warcraft_category:0 "Warcraft"
\ No newline at end of file
+ game_rule_category_warcraft_category:0 "Warcraft"
+
+ rule_wc_theramore_mechanic:0 "#clickable #Bold Warcraft#!#!: Theramore Mechanic#!"
+ setting_wc_theramore_mechanic_on:0 "#high On#!"
+ setting_wc_theramore_mechanic_off:0 "Off"
+ setting_wc_theramore_mechanic_on_desc:0 "Theramorean culture has a special mechanic with quests and settlers"
```
Comment (arithon) [inline localization/english/wc_game_rules_l_english.yml:61]:
```suggestion
 setting_wc_theramore_mechanic_on_desc:0 "The theramorean culture has a special mechanic with quests and settlers"
```

File: localization/english/wc_game_rules_l_english.yml [hunk 2]
```diff
@@ -53,4 +53,10 @@
...
+ setting_wc_theramore_mechanic_off_desc:0 "Theramorean culture doesn't have any special mechanic"
```
Comment (arithon) [inline localization/english/wc_game_rules_l_english.yml:62]:
```suggestion
 setting_wc_theramore_mechanic_off_desc:0 "The theramorean culture doesn't have any special mechanic"
```

**Review by arithon — CHANGES_REQUESTED** [review]
only left suggestions that can be implemented directly, otherwise all looks fine to me.

File: localization/english/gui/wc_theramore_mechanic_l_english.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+ government_te_name:0 "[GetPlayer.GetTopLiege.GetPrimaryTitle.GetNameNoTierNoTooltip] Government"
+ te_tasks_name:0 "Tasks"
+ policy_name_unions:0 "Unions Policy"
+ policy_name_trade:0 "Trade Policy"
+ policy_name_tolerance_culture:0 "Cultural Tolerance Policy"
+ policy_name_tolerance_faith:0 "Faith Tolerance Policy"
+ te_party_xenophilic:0 "Xenophilic Party"
+ te_party_xenophobic:0 "Xenophobic Party"
+ te_join_party:0 "Join this party"
+ settlers_plus_10_percent:0 "Settlers: #P +10%#! per year"
```
[the remark this answers is not in the document]
Comment (sumankun) [reply localization/english/gui/wc_theramore_mechanic_l_english.yml:34]: `Settlers per year #P +10%#! `
maybe like this then, other examples has numbers in the end

[the remark this answers is not in the document]
Comment (arithon) [reply localization/english/gui/wc_theramore_mechanic_l_english.yml:34]: Sounds good

File: localization/english/gui/wc_theramore_mechanic_l_english.yml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+ te_improve_control_te_name:0 "Improve control up to #L 100#! in"
+ te_do_not_attack_te_name:0 "Do not start a war as attacker for 10 years"
+ te_create_ally_te_name:0 "Create alliance with "
+ te_improve_opinion_te_name:0 "Improve opinion up to #L 100#! of"
+ te_make_a_child_te_name:0 "Have a child"
+ te_join_horde_te_name:0 "Join #L The Horde#! union"
+ te_leave_alliance_te_name:0 "Leave #L The Alliance#! union"
+ te_remove_policy_te_name:0 "Change the value of "
+ te_kill_the_person_te_name:0 "Kill and rid this world of "
+ te_learn_language_te_name:0 "Learn the next language "
```
[the remark this answers is not in the document]
Comment (arithon) [reply localization/english/gui/wc_theramore_mechanic_l_english.yml:90]: Hmm, maybe then like this?
```suggestion
 te_learn_language_te_name:0 "Learn the language:"
```

**Review by arithon — APPROVED** [review]
loc LGTM

**Pull request merged**